app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -22,7 +22,7 @@
   </div>
 
   <div class="col-md-2">
-    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today %>
+    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today,  required: "required" %>
```

Comment:
Actually, come to think of it, this touches on an open question we have on form validation that I'd rather not take action on at this time. Let's punt on this for now.